robyn/robyn.pyi

```diff
@@ -1,13 +1,17 @@
 from __future__ import annotations
 
-from typing import Callable, Optional, Tuple
+from typing import Callable, Optional
 
 class SocketHeld:
     def __init__(self, url: str, port: int):
         pass
     def try_clone(self) -> SocketHeld:
         pass
 
+class FunctionInfo:
```

Comment:
@AntoineRR , I think we will benefit more from making it a `dataclass`https://www.pythontutorial.net/python-oop/python-dataclass/

Reply:
Good idea yes! I'm adding it